docs/gather_data.rst

```diff
@@ -60,6 +60,91 @@ To gather data for the last N days or N hours, use ``--since``:
     $ # see data for the last 12 hours
     $ ./multivac/gather_job_data.py --since 12h
 
+Writing data to JSON file
+-------------------------
+
+To write data to JSON file, use ``--format json`` option. If you add `-t` option,
+script will write data about tests too:
+
+.. code-block:: console
+
+    {
+    "8374189766": {
+    "job_id": 8374189766,
+    "workflow_run_id": 3061077644,
+    "job_name": "out_of_source",
+    "branch": "master",
+    "commit_sha": "416500fed508968d6d890eb3ec3620ef954d1d0a",
+    "conclusion": "success",
+    "queued_at": "2022-09-15T14:02:44Z",
+    "started_at": "2022-09-15T14:02:56Z",
+    "completed_at": "2022-09-15T14:08:45Z",
+    "platform": "amd64",
+    "runner_label": [
+      "ubuntu-20.04-self-hosted"
+    ],
+    "runner_name": "ghacts-shared-8-16-n17",
+    "runner_version": "2.296.2"
+    "tests": [
+    {
+      "name": "box/tx_man.test.lua",
+      "configuration": null,
+      "status": "fail",
+      "test_number": 0
+    },
+    {
+      "name": "replication/qsync_advanced.test.lua",
+      "configuration": "memtx",
+      "status": "fail",
+      "test_number": 1
+    }]}}
+
+Writing data to InfluxDB
+------------------------
+
+To write data to InfluxDB, you need to set credentials as environmental
+variables (see `.env-example` file). You can add them to the `.env` file and
```

Comment:
```suggestion
variables (see the ``.env-example`` file). You can add them to the ``.env`` file and
```

Reply:
User can just set environmental variables in any other way, in the script we don't use `.env` file, just call the environmental variables